qml/silica4felgo/TextSwitch.qml

```diff
@@ -19,20 +19,22 @@ MouseArea {
     AppText {
         id: appText
         font.pixelSize: BVApp.Theme.fontSizeSmall
-        anchors.verticalCenter: parent.verticalCenter
+
         anchors.left: parent.left
         anchors.right: swico.left
+        anchors.bottom: parent.bottom
 
         anchors.leftMargin: BVApp.Theme.horizontalPageMargin + BVApp.Theme.sectionHeaderIconLeftPadding  + BVApp.Theme.fontSizeSmall * 1.05 + BVApp.Theme.sectionHeaderIconTextPadding
-
     }
 
     AppSwitch {
-      id: swico
```

Comment:
Please fix indentation, otherwise the diff becomes harder to read.

Reply:
Yep. That was the idea. But you are right --- I split it into two commits to demystify the review.